mux.go

```diff
@@ -0,0 +1,48 @@
+package sm_yamux
+
+import (
+	"io/ioutil"
+	"net"
+
+	mux "github.com/libp2p/go-libp2p-core/mux"
+	yamux "github.com/libp2p/go-yamux"
+)
+
+var DefaultTransport *Multiplexer
```

Comment:
Not sure about changing DefaultTransport name to DefaultMultiplexer, cause it may break ipfs, lotus, etc.